src/trackers/uniqueKeysTracker.ts

```diff
@@ -51,17 +47,39 @@ export function uniqueKeysTrackerFactory(
       
       if (uniqueTrackerSize >= cacheSize) {
         log.warn(`${LOG_PREFIX_UNIQUE_KEYS_TRACKER}The UniqueKeysTracker size reached the maximum limit`);
-        senderAdapter && senderAdapter.recordUniqueKeys(uniqueKeysTracker);
+        // @TODO trigger event to submitter to send mtk
         uniqueTrackerSize = 0;
       }
     },
     
-    start(): void {
-      // @TODO
+    /**
+     * Pop the collected data, used as payload for posting.
+     */
+    pop(toMerge?: { [featureName: string]: ISet<string> }) {
+      const data = uniqueKeysTracker;
+      uniqueKeysTracker = {};
+      if (toMerge) {
+        Object.keys(data).forEach((key) => {
+          if (toMerge[key]) toMerge[key] = {...toMerge[key],...data[key]};
```

Comment:
I am not sure if the spread operator will work with `SetPoly` (the Set implementation used on old browser that doesn't have native Set support).

We should test it, or we can simplify the code by removing the `toMerge` param for now. I think removing it is fine, but you should check with the team what is the expected behaviour if uniqueKeys post fails.